guides/tutorial/part-2/index.md

```diff
@@ -1,17 +1,15 @@
-<!-- Heads up! This is a generated file, do not edit directly. You can find the source at https://github.com/ember-learn/super-rentals-tutorial/blob/master/src/markdown/tutorial/part-2/index.md -->
+Super, vous avez atteint la seconde partie du tutoriel&nbsp;! Dans les sections suivantes, nous continuerons à construire sur la base des concepts clés appris dans la première partie.
 
-Hooray, you've made it to the second part of the tutorial! In the following sections, we'll build on the core concepts that we learned in the first part of the tutorial.
+En cours de route, nous ajouterons de nouvelles fonctionnalités à notre app "Super Rentals". À la fin de cette section, vous aurez implémenté une fonction de recherche et refactoré une bonne partie du code pour utiliser de nouveaux concepts Ember.
```

Comment:
Everywhere in this file "we will" is kept as "nous ...", it would make sense to keep this here as well.

```suggestion
En cours de route, nous ajouterons de nouvelles fonctionnalités à notre app "Super Rentals". À la fin de cette section, nous aurons implémenté une fonction de recherche et réarrangé une bonne partie du code pour utiliser de nouveaux concepts Ember.
```

Reply:
Yes, that one is just a careless mistake, I didn't do it on purpose, thanks for catching